sdk/halo-sdk/src/main/java/com/mobgen/halo/android/sdk/core/threading/HaloInteractorExecutor.java

```diff
@@ -154,6 +155,35 @@ public final ICancellable execute() {
     }
 
     /**
+     * Executes given operation into the thread manager in the same thread given an inline response.
```

Comment:
+     * Executes the given operation into the thread manager in the same thread providing an inline response.